app/src/ui/merge-conflicts/merge-conflicts-dialog.tsx

```diff
@@ -178,8 +177,21 @@ export class MergeConflictsDialog extends React.Component<
     conflictStatus: ConflictFileStatus,
     onOpenEditorClick: () => void
   ): JSX.Element | null {
-    switch (conflictStatus.kind) {
-      case 'text':
+    let content = null
+    if (conflictStatus.kind === 'text') {
+      if (conflictStatus.conflictMarkerCount === null) {
+        content = (
+          <li className="unmerged-file-status-conflicts">
+            <Octicon symbol={OcticonSymbol.fileCode} className="file-octicon" />
```

Comment:
With the patch to change how this path is detected, I see that we didn't cleanup the duplicated DOM here and we see two icons:

<img width="534" alt="screen shot 2018-11-09 at 10 24 11 am" src="https://user-images.githubusercontent.com/359239/48267832-ba864380-e409-11e8-866d-7a5572dc6832.png">